src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -330,4 +337,9 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     const res = await axios.post(url, evalscript, requestConfig);
     return res.data;
   }
+
+  public async updateLayerFromServiceIfNeeded(): Promise<void> {
```

Comment:
`S1GRDAWSEULayer.ts` extends `AbstractSentinelHubV3Layer` and already has `updateLayerFromServiceIfNeeded()` so it would override it.

Reply:
Should be fixed.

Reply:
Resolving.